src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="#logo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo, showing the company name in blue and white"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav-menu">
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#laptops&computers"
+              class="nav-link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#gadgets"
+              class="nav-link"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#tablets"
+              class="nav-link"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#photo"
+              class="nav-link"
+            >
+              photo
+            </a>
+          </li>
+
+          <li>
+            <a
+              href="#video"
+              class="nav-link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting should have a 2-space offset. Also, avoid adding blank lines between parent and child elements.